### PR TITLE
Docs-1072 Update tcwhatis.md

### PR DESCRIPTION
**Pull request by q5sys**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/en/truecommand/overview/tcwhatis.md [hunk 1]
```diff
@@ -4,9 +4,51 @@ description: "Describing the features and some use cases of the TrueCommand NAS
...
 <br><br>
+
+### What features does TrueCommand have?
+
+- Multiple deployment options: TrueCommand is a lightweight application that supports deployments in either a Virtual Machine or a Docker Container. TrueCommand Cloud is also available as a cloud-based subscription option that allows you to offload TrueCommand resources and deployment and only focus on fine-tuning your configuration.
+
+- NAS Fleet Dashboard: The TrueCommand dashboard provides visibility to an organization’s entire TrueNAS fleet. TrueCommand includes an auto-discovery tool that expedites identifying and integrating systems into TrueCommand.
+
+- Single Sign-on to all NAS Units
+ Once a TrueNAS system has been connected to TrueCommand, authorized administrators can quickly log in to that system through TrueCommand. This allows for quicker and simpler signons instead of looking up IP addresses and login credentials.  This is even more beneficial when using different secure passwords for each TrueNAS instance instead of a single password across multiple systems.
```
Comment (ericbsd) [inline content/en/truecommand/overview/tcwhatis.md:23]: Should "TrueCommand, authorized"  be "TrueCommand, and authorized"

File: content/en/truecommand/overview/tcwhatis.md [hunk 2]
```diff
@@ -4,9 +4,51 @@ description: "Describing the features and some use cases of the TrueCommand NAS
 weight: 10
 ---
 
-TrueCommand is a single Pane-of-Glass management application that takes the repetitive work out of multi-TrueNAS management by centralizing system alerts, reports, and analytics in one easy to use interface.
-It supports users and teams with 24x365 global operations, role-based access control, and a full log of every TrueNAS configuration change.
-TrueCommand runs on docker, VMs, or as a cloud service and is free to users with less than 50 drives and affordable for those with larger installations.
+### What is TrueCommand?
 
-<img src="/images/truecommand-overview.png">
+TrueCommand is a multi-system management "Single pane of Glass" system that helps control and monitor your TrueNAS fleet. TrueCommand assists in managing TrueNAS systems through REST APIs, WebSocket APIs, and a web user interface. The TrueCommand web interface provides unified administration of users and TrueNAS systems and single sign-on functionality.
```
Comment (ericbsd) [inline content/en/truecommand/overview/tcwhatis.md:9]: should "provides unified" be   "provides a unified"?

File: content/en/truecommand/overview/tcwhatis.md [hunk 3]
```diff
@@ -4,9 +4,51 @@ description: "Describing the features and some use cases of the TrueCommand NAS
...
+ TrueCommand records all administration actions in secure audit logs. This allows for quick identification of what has been changed and who changed it.
+
+
+### What does it cost?
+
+TrueCommand software is free to use to manage up to 50 drives. It can be deployed as a Docker container or as a VM. Licenses to expand TrueCommand capabilities may be purchased from the [iXsystems account portal](http://portal.ixsystems.com/). Pricing is based on the number of drives and the support level required.
+
+### Where do I get it?
+
+TrueCommand can be downloaded from the [TrueNAS website](https://www.truenas.com/truecommand/). TrueCommand Cloud subscriptions are available at the [iXsystems Account Services Portal](https://portal.ixsystems.com/portal/login/index.php#login)
```
Comment (ericbsd) [inline content/en/truecommand/overview/tcwhatis.md:44]: Missing a dot at the end.

**Review by ericbsd — CHANGES_REQUESTED** [review]

File: content/en/truecommand/overview/tcwhatis.md (hunk 1 above)
Comment (q5sys) [reply content/en/truecommand/overview/tcwhatis.md]: Not in this instance.  English grammar can be weird.  "Once a TrueNAS system has been connected to TrueCommand" is an introductory clause, so 'and' would be wrong.   But for ease of translation later I probably should re-write that sentence to be simpler.

File: content/en/truecommand/overview/tcwhatis.md (hunk 2 above)
Comment (q5sys) [reply content/en/truecommand/overview/tcwhatis.md:9]: yup, i missed the 'a'

Comment (q5sys) [reply content/en/truecommand/overview/tcwhatis.md]: This is another, #english_grammar_is_weird moment.  I've changed the sentence around a bit.

File: content/en/truecommand/overview/tcwhatis.md (hunk 3 above)
Comment (q5sys) [reply content/en/truecommand/overview/tcwhatis.md]: fixed

File: content/en/truecommand/overview/tcwhatis.md (hunk 1 above)
Comment (ericbsd) [reply content/en/truecommand/overview/tcwhatis.md]: The new version is way better.

**Review by ericbsd — APPROVED** [review]
LGTM

**Pull request merged**